Description: Using the group plugdev instead of usb
Author: Ruben Undheim <ruben.undheim@gmail.com>
Forwarded: <no|not-needed|url proving that it has been forwarded>

--- ubertooth-20141130git241874e.orig/host/libubertooth/40-ubertooth.rules
+++ ubertooth-20141130git241874e/host/libubertooth/40-ubertooth.rules
@@ -1,5 +1,5 @@
-ATTRS{idVendor}=="ffff", ATTRS{idProduct}=="0004", MODE="0660", GROUP="usb"
-ATTRS{idVendor}=="1d50", ATTRS{idProduct}=="6000", MODE="0660", GROUP="usb"
-ATTRS{idVendor}=="1d50", ATTRS{idProduct}=="6001", MODE="0660", GROUP="usb"
-ATTRS{idVendor}=="1d50", ATTRS{idProduct}=="6002", MODE="0660", GROUP="usb"
-ATTRS{idVendor}=="1d50", ATTRS{idProduct}=="6003", MODE="0660", GROUP="usb"
+ATTRS{idVendor}=="ffff", ATTRS{idProduct}=="0004", MODE="0660", GROUP="plugdev"
+ATTRS{idVendor}=="1d50", ATTRS{idProduct}=="6000", MODE="0660", GROUP="plugdev"
+ATTRS{idVendor}=="1d50", ATTRS{idProduct}=="6001", MODE="0660", GROUP="plugdev"
+ATTRS{idVendor}=="1d50", ATTRS{idProduct}=="6002", MODE="0660", GROUP="plugdev"
+ATTRS{idVendor}=="1d50", ATTRS{idProduct}=="6003", MODE="0660", GROUP="plugdev"
